controllers/propagator/policy_controller.go

```diff
@@ -122,20 +123,63 @@ func (r *PolicyReconciler) Reconcile(ctx context.Context, request ctrl.Request)
 				return reconcile.Result{}, err
 			}
 
+			if len(replicatedPlcList.Items) == 0 {
+				log.V(2).Info("No replicated policies to delete.")
+
+				return reconcile.Result{}, nil
+			}
+
+			policiesChan := make(chan policiesv1.Policy, len(replicatedPlcList.Items))
```

Comment:
Could you add an info log above this stating that the N replicated policies are being deleted because the root policy was deleted?

This helps us know what the reconcile method is doing since it could take a while to delete thousands of replicated policies.